### PR TITLE
Add Spawn At Node in Screw Attack Room

### PR DESCRIPTION
**Pull request by kjbranch**
This removes node 5, which was not useful since there was an obstacle. It then adds a spawn at node and node on the bomb blocks.

It looked like the heat frames throughout were a bit inconsistently determined (prepared through the door vs fall first). I did my best to separate them, but I'm sure the heat frames will need to be adjusted throughout.

### CODE REVIEW

File: region/lowernorfair/west/Screw Attack Room.json [hunk 1]
```diff
@@ -955,55 +914,77 @@
...
         ]},
         "h_canUseMorphBombs",
-        {"heatFrames": 300}
+        {"heatFrames": 220}
       ],
       "clearsObstacles": ["B"]
     },
     {
-      "link": [4, 2],
+      "link": [4, 5],
```
Comment (osse101) [inline region/lowernorfair/west/Screw Attack Room.json:981]: This would be able to land at 2

**Review by osse101 — COMMENTED** [review]
There is a [3, 2] canResetFallSpeed strat

File: region/lowernorfair/west/Screw Attack Room.json [hunk 2]
```diff
@@ -727,97 +705,34 @@
...
+        {"doorUnlockedAtNode": 2},
+        "canResetFallSpeed",
+        "canTrickyJump",
+        {"heatFrames": 145}
       ],
-      "note": [
-        "Moonfall to clip through the Bomb blocks",
-        "Break spin with Shot or Angle Aim to avoid clipping all the way to the bottom of the room."
+      "unlocksDoors": [
+        {"types": ["super"], "requires": ["canPrepareForNextRoom"]},
```
Comment (osse101) [inline region/lowernorfair/west/Screw Attack Room.json:734]: ?  Seems unrelated to anything

Comment (kjbranch) [reply region/lowernorfair/west/Screw Attack Room.json:734]: Ah, I put this on as you'd need to know that the door is green beforehand. If you've never been in the room or have the map and it was green, I don't think you could have enough time to react and would instead fall to 5

Comment (osse101) [reply region/lowernorfair/west/Screw Attack Room.json:734]: That makes sense 🧠
